HW_3/task_2.py

```diff
@@ -0,0 +1,25 @@
+"""
+Задача 6.
+Пользователь вводит строку текста. Вывести каждое слово с новой строки.
+? Строки нумеруются начиная с единицы.
+? Слова выводятся отсортированными согласно кодировки Unicode.
+? Текст выравнивается по правому краю так, чтобы у самого длинного
+слова был один пробел между ним и номером строки.
+"""
+
+
+def sorted_words():
+    SPACE = 1
+    data = input("Enter something: ")
+    lst = data.split()
+    lst.sort()
+    maxi = 0
+    for item in lst:
+        if len(item) > maxi:
+            maxi = len(item)
```

Comment:
Длину самого длинного слова можно найти гораздо проще, используя встроенные агрегирующие функции. Например так
```suggestion
    maxi = len(max(lst, key=len))
```